copier/main.py

```diff
@@ -426,8 +426,16 @@ def _render_file(self, src_abspath: Path) -> None:
         if dst_relpath is None:
             return
         if src_abspath.name.endswith(self.template.templates_suffix):
-            tpl = self.jinja_env.get_template(str(src_relpath))
-            new_content = tpl.render(**self._render_context()).encode()
+            try:
+                tpl = self.jinja_env.get_template(src_relpath)
+            except UnicodeDecodeError:
+                if self.template.templates_suffix:
+                    # suffix is not emtpy, re-raise
+                    raise
```

Comment:
Maybe we should raise a UserMessageError then?

Reply:
I think it's OK as it is. That kind of exception would print a nice human-readable message in the CLI, but in this case I don't think we should hide the stack trace. If you add the `.jinja` suffix to a binary file, it's a wrong usage of Copier and it should die badly, and the stack trace should tell you what's happening.

Let's see, if too many people open issues about that message not being understood, I think we can add it, but TBH I think it's too much of a corner case.